salt/metalk8s/addons/prometheus-operator/deployed/service-configuration.sls

```diff
@@ -0,0 +1,83 @@
+include:
+  - .namespace
+
+{%- set grafana_config = salt.metalk8s_kubernetes.get_object(
+        kind='ConfigMap',
+        apiVersion='v1',
+        namespace='metalk8s-monitoring',
+        name='metalk8s-grafana-config'
+  )
+%}
+
+{%- set prometheus_config = salt.metalk8s_kubernetes.get_object(
+        kind='ConfigMap',
+        apiVersion='v1',
+        namespace='metalk8s-monitoring',
+        name='metalk8s-prometheus-config'
+  )
+%}
+
+{%- set alertmanager_config = salt.metalk8s_kubernetes.get_object(
+        kind='ConfigMap',
+        apiVersion='v1',
+        namespace='metalk8s-monitoring',
+        name='metalk8s-alertmanager-config',
+  )
+%}
+
+{%- if grafana_config is none %}
+
+Create grafana-config ConfigMap:
+  metalk8s_kubernetes.object_present:
+    - manifest:
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          name: metalk8s-grafana-config
+          namespace: metalk8s-monitoring
+        data:
+          config.yaml: |-
+            apiVersion: addons.metalk8s.scality.com
+            kind: GrafanaConfig
+            spec:
+              deployment:
+                replicas: 1
+{%- endif %}
```

Comment:
An sls cannot be empty or you will get an error (or maybe just a warning don't remember) and here if all config already exists after jinja rendering you will get empty sls.

So suggest to just add an `else` block with `test.succeed_without_changes` explaining that the configmap already exists nothing need to be done